package-configuration/linux-cli/test_get_cloudify.py

```diff
@@ -144,6 +147,25 @@ def test_check_cloudify_installed_in_venv(self):
         finally:
             shutil.rmtree(tmp_venv)
 
+    def test_get_requirements_from_source_url(self):
+        installer = get_cloudify.CloudifyInstaller()
+        req_list = installer._get_default_requirement_files(cloudify_cli_url)
+        self.assertEquals(len(req_list), 1)
+        self.assertIn('dev-requirements.txt', req_list[0])
```

Comment:
I don't want to open up a fileserver here just so that it downloads it. For the local test, it's good.
This is actually a good test if not using 3.2 but rather master as it's always supposed to succeed. If it doesn't, it means something is wrong anyway. The best thing I can think of for an offline state is to ignore this test but I don't see this as a serious problem.